src/main/resources/liquibase/3.1.x.x/provisional_clinical_diagnosis.xml

```diff
@@ -0,0 +1,16 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
```

Comment:
I'm not sure this aligns with the overall formatting.

Reply:
@HerbertYiga can you correct the formatting here